.devcontainer/dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.9
+
+RUN --mount=type=bind,target=/requirements,type=bind,source=requirements <<EOF
```

Comment:
Let's build the image in the CI then.

Reply:
In a `build.image.yml`, with rules such that the action runs only if the `dockerfile` or `requirements/*` changed.

Reply:
Done. Cleaner indeed :sparkles: